plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java

```diff
@@ -198,7 +198,7 @@ static PermissionCollection getRestrictedPermissions() {
         Permissions perms = new Permissions();
         // property/env access needed for parsing
         perms.add(new PropertyPermission("*", "read"));
-        perms.add(new RuntimePermission("getenv.TIKA_CONFIG"));
+        perms.add(new RuntimePermission("getenv.*"));
```

Comment:
@reta I had to make this change to resolve the JSM permissions issue. I can narrow this down if needed, but thought wildcard was reasonable. lmk what you think.

Reply:
Since this same perm wouldn't be enforced in 3.x, I thought it was reasonable to use wildcard here.